helm/aws/basic-eks/values-result.yaml

```diff
@@ -21,6 +21,12 @@ deploy:
     period: "4"
     timeout: "2"
 
+ingress:
+  domain: codepipes.tk
+  certificateArn: arn:aws:acm:us-east-1:543332486884:certificate/df272f52-72fe-47ac-a4d3-9e07ec946175
+  appDomain: voting.codepipes.tk
```

Comment:
i don't think this should be hardcoded in this template - i.e. this could be used by anyone. Can we make it an input?

Reply:
These are just the sample values file that I use to test Helm Charts and is useful in debugging. The actual values.yaml is generated by our template on the fly and values are passed from there.